saleor/graphql/schema.graphql

```diff
@@ -2146,10 +2146,10 @@ input MoveProductInput {
 }
 
 type Mutation {
-  wishlistAddProduct(productId: ID!): WishlistAddProductMutation
-  wishlistRemoveProduct(productId: ID!): WishlistRemoveProductMutation
-  wishlistAddVariant(variantId: ID!): WishlistAddProductVariantMutation
-  wishlistRemoveVariant(variantId: ID!): WishlistRemoveProductVariantMutation
+  wishlistAddVariant(variantId: ID!): WishlistAddVariant
```

Comment:
I'm wondering whether to rename `variantId` to just `id` since it seems obvious from the mutation name that we need a variant ID here.

Reply:
@maarcingebala +1

Reply:
We should support adding multiple variantId`s at once(user is logged off -> adds lots of stuff to wishlist -> logs in -> we use mutation only once to add all stuff)

Reply:
So maybe we should have a list in all of these mutations for consistent API.